tests/lib/rules/no-this-translate.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @fileoverview Disallow importing from the root Lodash module
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require( '../../../lib/rules/no-this-translate' ),
+	RuleTester = require( 'eslint' ).RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+( new RuleTester() ).run( 'no-this-translate', rule, {
+	valid: [
+		'i8n.translate(\'hello\')',
```

Comment:
Minor: Typo on `i18n`

Should we also include `this.props.translate` as a valid example?